packages/components/pie-tag/src/tag.scss

```diff
@@ -80,6 +81,14 @@
         }
     }
 
+    &.c-tag--icon-placement--leading {
```

Comment:
Out of interest why is this empty?

Reply:
because the styles are the same as the default styles. 
it's a common approach in the codebase currently where the default variant of a prop is declared in the common styles so the styles are still applied if a user passes an invalid prop value.